lib/twilio.rb

```diff
@@ -0,0 +1,21 @@
+require 'rubygems'
+require 'twilio-ruby'
+require 'dotenv'
+Dotenv.load
+
+class TwilioSMS
+
+  def initialize( sms_messenger: Twilio::REST::Client.new((ENV['ACCOUNT_SID']), (ENV['AUTH_TOKEN'])))
```

Comment:
Metrics/LineLength: Line is too long. [101/80] (https://github.com/bbatsov/ruby-style-guide#80-character-limits)